cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
+
+    # Mark 32 bit executables large address aware so they can use > 2GB address space
```

Comment:
I'd appreciate feedback from Kathleen (or others) on this.

Unfortunately, the only version of Visual Studio that I had access to is pretty old -- VS 2010, and doesn't have 64-bit or ``C++11`` capabilities.

I think I can get an express version working the next time I have time to look into this.

However, (I think) the free version is missing some features like folders support, that I was hoping to play with.